BUILD.bazel

```diff
@@ -14,22 +12,34 @@ npm_link_all_packages(name = "node_modules")
 # Link the acorn package, which was fetched separately with npm_import from /WORKSPACE, to the
 # virtual store in bazel-bin/node_modules/.aspect_rules_js and as a direct link to
 # bazel-bin/node_modules/acorn
-npm_link_acorn(name = "node_modules/acorn")
+# buildifier: disable=duplicated-name
+npm_link_acorn(name = "node_modules")
+
+# Do the same for the pnpm package
+# buildifier: disable=duplicated-name
+npm_link_pnpm(name = "node_modules")
```

Comment:
hmm this is pretty confusing to have two rules named the same. I think the editor plugin and maybe buildifier also will yell at you that you've made a mistake

Reply:
fixed up